### ISSUE TITLE
Feature: Add the 'always-on-top' option

### ISSUE THREAD

**Issue by 13OnTheCode** [azat-io/eslint-plugin-perfectionist#29]
### What rule do you want to change?

sort-interfaces, sort-object-types

### Describe the problem

```
interface ButtonProps {
  children?: string | number
  className?: string
  onClick?: () => void
  onChange?: () => void
  size: 's' | 'm' | 'l'
  type?: 'submit' | 'button'
  variant?: 'solid' | 'outline' | 'text'
}

type Props = {
  children?: string | number
  className?: string
  onClick?: () => void
  onChange?: () => void
  size: 's' | 'm' | 'l'
  type?: 'submit' | 'button'
  variant?: 'solid' | 'outline' | 'text'
}
```

```
{
  "plugins": ["perfectionist"],
  "rules": {
    "perfectionist/sort-interfaces": [
      "error",
      {
        "type": "natural",
        "order": "asc",
        "always-on-top": ["on*"]
      }
    ],
   "perfectionist/sort-object-types": [
      "error",
      {
        "type": "natural",
        "order": "asc",
        "always-on-top": ["on*"]
      }
    ]
  }
}
```

Expected Result:

```
interface ButtonProps {
  onClick?: () => void
  onChange?: () => void
  children?: string | number
  className?: string
  size: 's' | 'm' | 'l'
  type?: 'submit' | 'button'
  variant?: 'solid' | 'outline' | 'text'
}

type Props = {
  onClick?: () => void
  onChange?: () => void
  children?: string | number
  className?: string
  size: 's' | 'm' | 'l'
  type?: 'submit' | 'button'
  variant?: 'solid' | 'outline' | 'text'
}
```

### Code example

...

### Validations

- [X] Read the [docs](https://eslint-plugin-perfectionist.azat.io).
- [X] Check that there isn't [already an issue](https://github.com/azat-io/eslint-plugin-perfectionist/issues) that reports the same bug to avoid creating a duplicate.

**Comment by azat-io** [comment]
That's a good idea. I think I can do it on Thursday

**Comment by azat-io** [comment]
Will be implemented in v2.0.0 (https://github.com/azat-io/eslint-plugin-perfectionist/issues/32)

**Comment by azat-io** [comment]
Added in 903c0292291a4109b1aeaa2e3c24afd2c9d605a6.

Release is coming soon.

**Comment by azat-io** [comment]
Released in [v2.0.0](https://github.com/azat-io/eslint-plugin-perfectionist/releases/tag/v2.0.0)
